### DISCUSSION TITLE
I am new, I have a doubt

### DISCUSSION THREAD

**Discussion by ghost** [hlky/stable-diffusion-webui#533] [Q&A]
I already know how to use it but I want to know if every time I use it I have to run all the commands every time? or is there another way?

**Comment by ReinerBforartists** [comment]
Not anymore. But some pre - steps are still required.

First step is not to clone the repo from the guide, which was /waifu-diffusion. Also not this development repo hlky/stable-diffusion.webui. But the hlky/stable-diffusion one. You have of course still to do the steps to download and place the required files into the needed folders. But once you are done it's only one click to start stable diffusion, and then open the webpage to create your images. This is all handled by the webui.cmd in the repository.

Kind regards
Reiner
